README.md

```diff
@@ -5,41 +5,29 @@ This project is currently under active development, and as such, all source code
 
-![figure 1. Intel Data Center GPU Driver Container Image Releasing Flow](pics/releasing_flow.jpg)
+Intel data center GPU driver container image is built from [Intel GPU Driver Repository](https://github.com/intel-gpu). It includes:
+-	[Intel® Graphics (i915) Driver Backports for Linux for RHEL](https://github.com/intel-gpu/intel-gpu-i915-backports/tree/redhat/main)
+-	[Intel® Converged Security Engine (CSE) Backports for RHEL-8.6](https://github.com/intel-gpu/intel-gpu-cse-backports/tree/rhel86)
+-	[Intel® Platform Monitoring Technology (PMT) Backports for RHEL-8.6](https://github.com/intel-gpu/intel-gpu-pmt-backports/tree/rhel86)
+-	[Intel® GPU firmware](https://github.com/intel-gpu/intel-gpu-firmware)
 
-**Intel Data Center GPU Driver for OpenShift project** is used to build, package, and release Intel data center GPU driver images for [Red Hat OpenShift Container Platform (OCP)](https://www.redhat.com/en/technologies/cloud-computing/openshift/container-platform). Users can make use of the [pre-built & released images](release/README.md) to provision Intel Data Center GPU Cards on OCP. They also can use Intel dGPU Driver Dockerfile released from this project as a reference to on-premises build the driver container images and provision the cards. In this 0.1.0 release [Intel Flex Serial GPU cards](https://www.intel.com/content/www/us/en/products/docs/discrete-gpus/data-center-gpu/flex-series/overview.html) are supported. [Intel Max Serial GPU card support](https://www.intel.com/content/www/us/en/products/details/discrete-gpus/data-center-gpu/max-series.html) is in the plan.
 
-Intel dGPU driver container image is built from [Intel GPU Drivers](https://github.com/intel-gpu) which includes
+To build Intel Data Center GPU driver images and provision the GPU cards on OCP, two scenarios have to be supported.
 
-*	**[Intel GPU i915 driver for RHEL](https://github.com/intel-gpu/intel-gpu-i915-backports)**
-* **[Intel® Converged Security Engine(CSE) driver for RHEL](https://github.com/intel-gpu/intel-gpu-cse-backports)**
-*	**[Intel® Platform Monitoring Technology(PMT) driver for RHEL](https://github.com/intel-gpu/intel-gpu-pmt-backports/)**
-*	**[Intel® GPU firmware](https://github.com/intel-gpu/intel-gpu-firmware)** 
+- **Driver Upgrade Scenario** - There is a new release from Intel GPU Drivers.In this scenario, Intel data center GPU driver Dockerfile needs to be updated to include the new release, Intel data center driver container image will be built and packaged from the Dockerfile. After testing and certifying steps, the new image should be released on Red Hat Ecosystem Catalog.    
```

Comment:
Add a space between "There is a new release from Intel GPU Drivers." and "In this scenario"